cypress/integration/search/post_search_display.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+
+describe('Post search display', () => {
+    it('M14252 After clearing search query, search options display', () => {
+        // 1. Login and navigate to the app
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        const searchWord = 'ok';
+
+        // 2.search word in searchbox
+        cy.get('#searchBox').type(searchWord).type('{enter}');
```

Comment:
> 1. Search for any random word you know will return results

Searching text of "ok" will not yield result if no "ok" was previously posted. 

I'd suggest to post a text/message first (``cy.postMessage('Hello');``), then search for the posted message.  After that, validate if there is "search result/s in the RHS"